api/app/signals/apps/signals/migrations/0109_merge_20200527_0951.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.12 on 2020-05-27 07:51
```

Comment:
Remove this after basing `0108_auto_20200525_1406` on `0108_area_uniqueness_constraints` --- this migration is not really needed.